.github/actions/next-stats-action/Dockerfile

```
@@ -1,13 +1,20 @@
-FROM node:16-bullseye
+FROM ubuntu:22.04
 
 LABEL com.github.actions.name="Next.js PR Stats"
 LABEL com.github.actions.description="Compares stats of a PR with the main branch"
 LABEL repository="https://github.com/vercel/next-stats-action"
 
 COPY . /next-stats
 
+RUN apt update && apt upgrade -y
+RUN apt install unzip wget curl nano htop screen build-essential pkg-config libssl-dev git build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libreadline-dev libffi-dev python3 moreutils jq iproute2 openssh-server sudo whois dnsutils -y
+
+RUN ln $(which python3) /usr/bin/python
+
+RUN curl -sfLS https://install-node.vercel.app/v18 | bash -s -- -f
+
 # Install node_modules
-RUN npm i -g pnpm@7.24.3
+RUN npm i -g pnpm@7.24.3 yarn
```

Might want to specific a version of yarn here too

```suggestion
RUN npm i -g pnpm@7.24.3 yarn@1.22.19
```